jenkins/automation/kubernetes/nightly-build-and-deploy.groovy

```diff
@@ -186,19 +186,39 @@ pipeline {
                 env.build_location = "${DOCKER_IMAGE_LOCATION}"
                 env.deployment_status = "${MESSAGE}"
                 env.cluster_status = "${env.build_setupcortx_url}"
+                env.cortx_script_branch = "${CORTX_SCRIPTS_BRANCH}"
```

Comment:
Email notification shows below - 
![image](https://user-images.githubusercontent.com/63289460/152933620-0a1d44be-3f72-40aa-a102-a494b52e8ea5.png)

Please change "Cortx Service Build" to "CORTX Services Release"

Reply:
okay